index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto:wght@700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+    <title>Portfolio</title>
+</head>
+<body>
+    <div class="status-bar">
+        <div class="clock">9:40</div>
+        <div class="status"><img src="Img/status.png" alt="wifi"></div>
+        <div class="name">Arturo Hermida</div>
+        <div class="dropdown"><img src="Img/Enabled.png" alt="drop"></div>
```

Comment:
- [ ] `Arturo Hermida` is supposed to be a logo, and usually, logo is a link that takes users to the homepage. Please wrap Arturo Hermida with  an `<a> `tag
- [ ] Please try to match your design with the one in Figma, currently logo on your design is wrapping and is not vertically centred :
 
Your design  | Figma design
------------- | -------------
<img width="375" alt="Screenshot 2022-03-17 at 10 17 31" src="https://user-images.githubusercontent.com/77038610/158748556-b6813170-8ce4-4574-8fb2-f14ece9d7d5a.png"> | <img width="387" alt="Screenshot 2022-03-17 at 10 17 51" src="https://user-images.githubusercontent.com/77038610/158748596-7a366e14-b3c1-44df-a85b-d6ac3ac62a11.png">
Please center the content, decrease the logo font size and make sure to have same paddings